### PR TITLE
Navigation (React)

### PR DESCRIPTION
**Pull request by marrep**
Header and Footer are build in and (almost) consistently present.
Initiate react app, setting up routing, creating components, assign paths and links.

### CODE REVIEW

File: frontend/src/index.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+body,
+html {
+  margin: 0 !important;
```
Comment (itsame1985) [inline frontend/src/index.css:3]: Maybe you could use Globalstyles.js?

**Review by itsame1985 — CHANGES_REQUESTED** [review]

File: frontend/src/components/header/HeaderStyled.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+export const HeaderBar = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 10vh;
+  background-color: #003f8a;
+`;
```
Comment (annikavennemann) [inline frontend/src/components/header/HeaderStyled.js:11]: Sieht gut aus :) Hast du das mit Styleguidist gemacht?

File: frontend/src/components/navigation/NavigationStyled.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  width: 100%;
+  height: 10vh;
+  bottom: 0;
+  position: absolute;
+  background-color: #f1f1f1;
+`;
```
Comment (annikavennemann) [inline frontend/src/components/navigation/NavigationStyled.js:13]: I would always set the "position:absolute" as the first attribute. So whenever you want to relate another element to this styled component, you see quickly what criteria you used here.

File: frontend/public/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (FabianAlleblas) [inline frontend/public/index.html:10]: You could add a description for your app.

File: frontend/public/index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (FabianAlleblas) [inline frontend/public/index.html:27]: and also a title

File: frontend/src/components/app/AppStyled.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+import styled from "styled-components";
+
+export const GlobalWrapper = styled.div`
+  margin: 0;
+  padding: 0;
+`;
```
Comment (FabianAlleblas) [inline frontend/src/components/app/AppStyled.js:6]: May you could make a globalstyle for resetting margin and padding

File: frontend/src/index.css (hunk 1 above)
Comment (FabianAlleblas) [reply frontend/src/index.css:3]: Would also recommend that, !important is not the best way.

File: frontend/src/components/navigation/Navigation.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { FooterWrapper } from "./NavigationStyled";
+import Bookmark from "../../assets/bookmark.svg";
+import Cart from "../../assets/cart.svg";
+import Home from "../../assets/home.svg";
+import Menu from "../../assets/menu.svg";
+import Search from "../../assets/search.svg";
```
Comment (FabianAlleblas) [inline frontend/src/components/navigation/Navigation.js:8]: You could build a Icon.js where you import all icons and can easy export them
`import {Bookmark, Cart, Home, Menu, Serach} from "../../Icon"`

**Review by FabianAlleblas — COMMENTED** [review]
Nice and easy PR, like it, wrote a few comments, hope they help.

File: frontend/src/components/header/HeaderStyled.js (hunk 2 above)
Comment (marrep) [reply frontend/src/components/header/HeaderStyled.js:11]: Ne, noch nicht :-)

File: frontend/src/components/navigation/Navigation.js (hunk 7 above)
Comment (marrep) [reply frontend/src/components/navigation/Navigation.js:8]: is there a typical folder to put this .js ?

Comment (annikavennemann) [reply frontend/src/components/navigation/Navigation.js:8]: I'd just call the folder "images" or even "icons" :)

**Pull request merged**